### ISSUE TITLE
Esperanto translation corrections

### ISSUE THREAD

**Issue by jorgesumle** [emesik/dont-buy-monero-sticker#21]
La teksto estas malĝusta, ĉar «Monero» estas nur unu ero, sed vi verŝajne volas aĉeti pli ol unu. Eĉ se vi nur volus aĉeti unu, ni devus aldoni la akuzativan n-finaĵon. Krome, oni diras "ĉifrovalutoj" kaj plej bonas "ŝtatan aparaton".

NE AĈETU MONEROJN

Ĉifrovalutoj estas damaĝemaj al la banksistemo kaj povas malfortigi la ŝtatan aparaton

**Comment by emesik** [comment]
We discussed it and decided not to translate MONERO if possible.

Regarding the second correction, is "aparaton de ŝtato" grammatically incorrect?

**Comment by jorgesumle** [comment]
> Regarding the second correction, is "aparaton de ŝtato" grammatically incorrect?

No, but it sounds weird, and it means something like the "apparatus of a state". It's like saying in English "apparatus of state". You can [see here](https://glosbe.com/en/eo/state%20apparatus) that people usually say "ŝtatan aparaton", and that's the best translation in my opinion.

![state apparatus](https://user-images.githubusercontent.com/14212780/195301601-86234605-9752-41b0-8a7d-9f181ff415e0.png)

**Comment by jorgesumle** [comment]
Another option that means the same is "Ĉifrovalutoj estas damaĝemaj al la banksistemo kaj povas malfortigi la **ŝtataparaton**".

**Comment by jorgesumle** [comment]
I opted for this translation:

![ne-aĉetu-Monerojn-glumarko](https://github.com/monero-ecosystem/dont-buy-monero-sticker/assets/14212780/8a3c85c1-1aac-4328-b2fc-4aef2bf5f11f)
